components/scrubber/config.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package scrubber
+
+import "regexp"
+
+var (
```

Comment:
Have you thought about how we avoid copy-pasting these around? We'll need these in some form or other in:

1. In this file
2. In a similar file for TypeScript
3. In Prometheus drop_label configs
4. In our fluent bit configuration (e.g. if se use [fluent/fluent-plugin-sanitizer](https://github.com/fluent/fluent-plugin-sanitizer))
5. Amazon Macie configuration

Figuring this out is not a blocker for the PR.